CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+- links now all connect to each other between STAC catalog, Datasets (?), and Items. STAC points to all Datasets, each Dataset points to items. items point to STAC catalog as 'root' and Dataset as 'parent'.
+
+### Changed
+- sat-api versioning from now matches versioning of STAC
```

Comment:
my concern with matching the STAC versioning is that how do handle changes on own software. We cannot issue major or minor updates anymore because we are stuck with the `0.6.0` for a while.

I suggest we have two separate versioning:

- one for the api endpoint which is on par with stac
- the other for the software which is separate from stac and can increment on its own

Reply:
@scisco Yes, this is a valid concern, and I had thought about it. However, the reason I decided to match versions at this point is because STAC versions are changing so rapidly. I think the sat-api versions will break away at some point, but this was a lot easier to track for right now. When we have a STAC version 1.0 we can cut a sat-api version 1.0 and then go our own way with versioning.

In the meantime, if we do need to increment the version, we can use the revision part, so issue a 0.6.1.  Although we have used the revision number, it's even more unlikely to be used by STAC going forward (since now we are using release candidates, and are planning for a 0.7.0 candidate by the end of year).